src/content/docs/dropins/checkout/tutorials/add-payment-method.mdx

```diff
@@ -9,131 +9,90 @@ import { Tabs, TabItem, Code, Steps } from '@astrojs/starlight/components';
 
 <Tasks>
 
 <Task>
 ### Prerequisites
 
-You must [install and configure](https://docs.adyen.com/plugins/adobe-commerce/set-up-the-plugin-in-adobe-commerce) the Adyen Payment Plugin on your Adobe Commerce backend before integrating it with the Commerce boilerplate template.
-
-<Aside type="note" title="Compatibility">
-The Adyen package is not compatible with [API Mesh](https://developer.adobe.com/graphql-mesh-gateway/).
-</Aside>
+You must [configure](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/payments/braintree) the Braintree extension on your Adobe Commerce backend before integrating it with the Commerce boilerplate template.
 
 </Task>
 
 <Task>
 
-### Add your client key
-
-After you install and configure the Adyen Payment Plugin on Adobe Commerce, you must add your Adyen client key to the Commerce boilerplate configuration.
+### Add the Braintree client SDK
 
-<Aside type="note">
-  You should use a development or staging environment for integration testing purposes before modifying your production configuration.
-</Aside>
+To integrate the Braintree payment provider with the Commerce boilerplate template, you must add the Braintree client SDK to the `commerce-checkout.js` file.
 
 <Steps>
-1. Open your project's `configs-stage` or `configs-dev` Google Sheet or Microsoft Excel file.
-
-1. Add a new key-value pair for your Adyen API key.
-
-   | key               | value           |
-   |-------------------|-----------------|
-   | adyen-client-key  | YOUR_CLIENT_KEY |
-
-1. Save the file.
-1. Publish file changes with the [Sidekick](/get-started/#sidekick) extension.
-   
-</Steps>
-
-</Task>
-
-<Task>
-
-### Install the package
+1. Open the `blocks/commerce-checkout/commerce-checkout.js` file.
 
-<Steps>
-1. Open a terminal and navigate to your boilerplate-based project.
-1. Install the Adyen NPM package and add it to the list of project dependencies in your `package.json` file with the `--save` flag.
+1. Import the Braintree client SDK.
    
-   ```bash
-   npm install @dropins/adyen-checkout-extension --save
+   ```js
+   ...
+   import 'https://js.braintreegateway.com/web/dropin/1.43.0/js/dropin.js';
+   ...
    ```
+
 </Steps>
 
 </Task>
 
 <Task>
 
-### Integrate with the boilerplate
+### Define a payment method modifier
 
-To integrate third-party payment providers, like Adyen, with Adobe Commerce, you can use the extensibility features provided by the checkout drop-in component. The checkout component allows you to customize the list of payment methods shown during the checkout process using slots.
+To render the Braintree payment method UI in the `PaymentMethods` container, you must define a modifier that you can use in the container's `main` slot.
```

Comment:
What is a `modifier`?

Reply:
@mcocapiza, this comes from the example code that you provided in https://github.com/commerce-docs/microsite-commerce-storefront/pull/107. Is "modifier" the correct term? If so, can you help define.